web-apps/genesys-spark-examples/package.json

```diff
@@ -55,7 +55,7 @@
     "stylelint-config-recess-order": "^6.0.0",
     "stylelint-config-standard": "^36.0.1",
     "stylelint-scss": "^6.11.1",
-    "webpack": "^5.97.1",
+    "webpack": "5.97.1",
```

Comment:
The next version of webpack introduced a types change that is causing a build error. We will have to investigate updating it separately.